.stylelintignore

```diff
@@ -0,0 +1,2 @@
+**/*.min.css
+test/functional/**/*.css
```

Comment:
Correct me if I'm wrong, but it seems like the contents of the `expected` folder are the only files that are generated programatically (when we expect a diff, we run `markbind build` on the test site and replace `expected` with the output).

Can we further restrict this to `test/functional/**/expected/**/*.css` to make it more specific? This way, the css files that are written by hand for the test sites can still be linted.

Reply:
Makes sense! Even though we currently don't have any hand-written CSS files for the test sites, this is still a good idea in case we decide to in the future.